### PR TITLE
Align to latest Digest I-D editorial conventions

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

aligns with the latest I-D.ietf-httpbis-digest-headers editorial conventions

### CODE REVIEW

**Review by msporny — APPROVED** [review]
Looks good @ioggstream, thank you! Can you point to the Digest spec text so I can double check that I'm looking at the right spec, please? 

I'm assuming (Oh, and look at who the author of that I-D currently is!) :) ... :

https://tools.ietf.org/html/draft-ietf-httpbis-digest-headers-03#section-5

```
All digest-algorithm values are case-insensitive but the lower case is preferred.
```

If so, I'll merge right after you confirm that I'm looking at the right spec.

**Comment by ioggstream** [comment]
You're looking at the right spec!

1- digest-algorithms have always been case-insensitive, so  RFC3230 allows using lowercase tokens
2- we just say that lowercase is preferred because some tokens like "CRC32c" may lead to think that tokens are case-sensitive

So this PR is ok with both RFC3230 and the new spec :)

**Comment by msporny** [comment]
Got it, thanks, merging!

**Pull request merged**
